backend/package.json

```diff
@@ -3,33 +3,15 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "ng": "ng",
-    "dev": "ng serve & yarn start_server",
     "start": "NODE_ENV=prod node server.js",
-    "build": "ng build --prod --aot",
-    "p": "yarn build && git add . && git commit --allow-empty -m \"latest build\" && git push origin master",
-    "test": "ng test",
     "startDatabase": "docker run -d --name buckets_db -p 27017:27017 -p 28017:28017 -e MONGODB_USER=\"username\" -e MONGODB_DATABASE=\"buckets_test\" -e MONGODB_PASS=\"password\" -e AUTH=no tutum/mongodb",
-    "lint": "ng lint",
-    "e2e": "ng e2e",
-    "build_server": "nodemon --exec 'ng build && node server.js'",
+    "build_server": "nodemon --exec 'node server.js'",
     "start_server": "NODE_ENV=dev nodemon --exec 'node server.js'",
     "test_server": "NODE_ENV=test nodemon --exec 'mocha --recursive'"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^4.2.3",
-    "@angular/common": "^4.2.3",
-    "@angular/compiler": "^4.2.3",
-    "@angular/compiler-cli": "^4.2.3",
-    "@angular/core": "^4.2.3",
-    "@angular/forms": "^4.2.3",
-    "@angular/http": "^4.2.3",
-    "@angular/material": "^2.0.0-beta.6",
-    "@angular/platform-browser": "^4.2.3",
-    "@angular/platform-browser-dynamic": "^4.2.3",
-    "@angular/router": "^4.2.3",
-    "body-parser": "^1.17.1",
+    "bluebird": "^3.5.0",
```

Comment:
okay.. and it's written in C. Bluebird is very proud of their code haha
https://softwareengineering.stackexchange.com/questions/278778/why-are-native-es6-promises-slower-and-more-memory-intensive-than-bluebird